python/ujson.c

```diff
@@ -53,7 +53,7 @@ PyObject* objToJSONFile(PyObject* self, PyObject *args, PyObject *kwargs);
 PyObject* JSONFileToObj(PyObject* self, PyObject *args, PyObject *kwargs);
 
 
-#define ENCODER_HELP_TEXT "Use ensure_ascii=false to output UTF-8. Pass in double_precision to alter the maximum digit precision of doubles. Set encode_html_chars=True to encode < > & as unicode escape sequences. Set escape_forward_slashes=False to prevent escaping / characters."
+#define ENCODER_HELP_TEXT "Use ensure_ascii=false to output UTF-8. Pass in double_precision to alter the maximum digit precision of doubles. Set encode_html_chars=True to encode < > & as unicode escape sequences. Set escape_forward_slashes=False to prevent escaping / characters. Set reject_bytes=True to raise TypeErro on bytes."
```

Comment:
Typo:
```suggestion
#define ENCODER_HELP_TEXT "Use ensure_ascii=false to output UTF-8. Pass in double_precision to alter the maximum digit precision of doubles. Set encode_html_chars=True to encode < > & as unicode escape sequences. Set escape_forward_slashes=False to prevent escaping / characters. Set reject_bytes=True to raise TypeError on bytes."
```